### ISSUE TITLE
Why only linear models? 

### ISSUE THREAD

**Issue by ezztahoun** [ustunb/actionable-recourse#15]
Did you experiment with non-linear models? If so what are the notes?

If not, why? What would be an approach to get recourse on more complex models (DNN) ?

**Comment by naveenmarthala** [comment]
yeah, if not DNNs, adding support for tree based models like RandomForest, Bagging, AdaBoot, XGBoost, LightGBM would be helpful too.
